docs/3.0/scopes.md

```diff
@@ -70,7 +70,7 @@ The scope you return here is going to be applied to the query of records on that
 You can use a symbol which will indicate the scope on that model or a block which will have the `query` available so you can apply any modifications you need.
 
 ```ruby
-class Avo::Scopes::EvenId < Avo::Pro::Scopes::BaseScope
```

Comment:
This is going to be a breaking change for everyone @Paul-Bob 

Let's add a class in Pro that inherits from the advanced one to ensure compatibility.

Also let's check the generators to have the proper default value.

Reply:
> This is going to be a breaking change for everyone

Addressed [here](https://docs.avohq.io/3.0/upgrade.html#scopes)

> Let's add a class in Pro that inherits from the advanced one to ensure compatibility.

`avo-pro` do not have access neither dependency on `avo-advanced`, are you sure we really want to add a dependency on it to be able to inherit?

> Also let's check the generators to have the proper default value.

Good catch, forgot about that one, already ficed